electron/execution.js

```diff
@@ -245,6 +245,7 @@ async function onTest(data, browserWindow) {
      */
     synthCliProcess = fork(`${SYNTHETICS_CLI}`, args, {
       env: {
+        ...process.env,
```

Comment:
This is something I'm uncertain about as I'm not accustomed to the security implications of changes to CLI interaction. From speaking offline it seems like this is not a large risk, but I'd like to hear others' opinions.

Reply:
As @emilioalvap mentioned in another comment, I think there will be various types of env values as well as platform specific ones, so it can be a bit tricky to track all of them. We should measure the effort vs worth by doing so (if we decide to do it). 

Another note, when env option is not specified when spawning, Node assigns parent's `process.env` as a [default](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options)